app/demo/myaccount/RecoverPasswordController.java

```diff
@@ -0,0 +1,8 @@
+package demo.myaccount;
```

Comment:
Would be nice to have the `demo` package for the app on master too. What was the reason to drop it on master?

Reply:
The package name was simply renamed, as the convention for Play applications is to have the controllers inside the `controllers` folder. In our case "demo" could made some sense, but once imported this to Starter we had to rename it to the convention because it was meant for a normal Play project.

Reply:
Ah, I forgot about this convention and thanks for your explanation 😄